Game.cpp

```diff
@@ -170,21 +180,25 @@ void Game::processInput()
                         loadNewShape();
                         break;
                 }
-                prepareScene();
-                std::cout << evnt.key.keysym.scancode << std::endl;
-
+                //std::cout << evnt.key.keysym.scancode << std::endl;
         }
     }
 }
 
 void Game::gameLoop()
 {
     loadNewShape();
-    prepareScene();
-    while (gameState != GameState::EXIT)
+    while (true)
     {
-        processInput();
+        prepareScene();
         presentScene();
-        SDL_Delay(16);
+        SDL_Delay(DELAY);
```

Comment:
This is currently how the timer is implemented: off the existing `SDL_DELAY` call.

It's a naive approach that assumes that the rest of the program takes no time at all to run.

In other words, it may run slower than anticipated and be vulnerable to slowdowns caused by getting input. 

I suppose, however, that also means the amount of input opportunities will at least be consistent.